README.md

```diff
@@ -52,6 +43,29 @@ For a video deep dive into this repo and the source code, please see the Microso
 
 [<img src="./docs/resources/deep-dive-video.jpg" width="50%">](https://youtu.be/PNykfuJ3VDs)
 
+### Azure Device Update
+
+You can interact with Azure Device Update, remotely updating your Azure IoT Hub devices through the samples below:
```

Comment:
Would it make sense to link to the readme here as well since it's kind of a porting guide for more devices? Just a suggestion, not sure if we tend to link across repos like that!

Reply:
I thought about that but was skeptical of having cross-repo links. But I do agree it would be good for customers, they would find the documentation more easily. I'll do it.

Reply:
Done.